yaml/boundaries.yaml

```diff
@@ -77,16 +77,20 @@ global:
           - geom_type: line
           - mz_boundary_from_polygon: true
 
-  - &dispute_id_components
+  - &osm_dispute_id_components
     tz_breakaway_code: { col: 'ne:brk_a3' }
     tz_ne_id: { col: 'ne_id' }
 
+  - &ne_dispute_id_components
+    tz_breakaway_code: { col: 'BRK_A3' }
```

Comment:
this should be lowercase, see my other comment above.

>    tz_breakaway_code: { col: 'brk_a3' }

Reply:
fixed in [4624bff](https://github.com/tilezen/vector-datasource/pull/2079/commits/4624bff282298e346af4fb4f60dda5f1d1b86a7d)